packages/aws-rfdk/lib/lambdas/nodejs/configure-spot-event-plugin/test/handler.test.ts

```diff
@@ -205,10 +205,22 @@ describe('SEPConfiguratorResource', () => {
     connection: validConnection,
   };
 
+  const deadlineGroups =  ['group_name'];
+  const deadlinePools =  ['pool_name'];
+
   const allConfigs: SEPConfiguratorResourceProps = {
     spotPluginConfigurations: validSpotEventPluginConfig,
     connection: validConnection,
     spotFleetRequestConfigurations: validSpotFleetRequestConfig,
+    deadlineGroups,
+    deadlinePools,
+  };
+
+  const noGroupsButPools: SEPConfiguratorResourceProps = {
```

Comment:
Instead of creating this variable up here, in the test we need to remove groups, we can do something like this instead:
```ts
const configNoGroups: SEPConfiguratorResourceProps = {
  ...allConfigs,
  deadlinePools: [],
}
```
This way we're keeping this test case input inside the test it's used in.

Reply:
`allConfigs` contains group, so your example just create config with empty Pools but with Groups. This is not what I need for test.

Reply:
I still think we can do away with this variable and just override `allConfigs` as necessary in each test. It might get pretty messy up here as we add more test cases to this file. But it's more of a nitpick, so resolving.